### DISCUSSION TITLE
Paint.net like Toolbox and Color Wheel

### DISCUSSION THREAD

**Discussion by AshleyPikachu** [PintaProject/Pinta#828] [Ideas]
Something like this would be great for work flow, the color wheel would be the most helpful.

![image](https://github.com/PintaProject/Pinta/assets/26305479/393e2e06-2202-4d2d-bf8b-92d2e4ab7996)

**Comment by cameronwhite** [comment]
Somewhat related to issues #570 and #761. The default color picker widget from GTK isn't great..
